bitnami/kube-prometheus/Chart.yaml

```
@@ -25,6 +25,7 @@ keywords:
   - alertmanager
   - operator
   - monitoring
+kubeVersion: ">= v1.7.0"
```

Please, use single quotes
```suggestion
kubeVersion: '>= 1.7.0'
```